pydrive2/test/test_util.py

```diff
@@ -1,9 +1,60 @@
+import unittest
 import random
 import re
+import os
+
+from funcy import retry
+from funcy.py3 import cat
+from pydrive2.files import ApiRequestError
 
 newline_pattern = re.compile(r'[\r\n]')
 
 
+class PyDriveTestBase(unittest.TestCase):
+    GDRIVE_USER_CREDENTIALS_DATA = "GDRIVE_USER_CREDENTIALS_DATA"
+    DEFAULT_USER_CREDENTIALS_FILE = "credentials/1.dat"
+    if os.getenv(GDRIVE_USER_CREDENTIALS_DATA):
```

Comment:
should it be part of setUp or something? it is strange to see so much code on the global class level

Reply:
right, have updated all automated TestCase classes to do setup in
```
@classmethod
  def setup_class(cls):
```